matrix-sdk/src/main/java/org/matrix/androidsdk/MXSession.java

```diff
@@ -741,6 +743,16 @@ public void removeMediasBefore(final Context context, final long timestamp) {
                                     filesToKeep.add(Uri.parse(mediaMessage.getUrl()).getPath());
                                 }
                             }
+                        } else if (TextUtils.equals(Event.EVENT_TYPE_STICKER, event.getType())) {
+                            StickerMessage stickerMessage = JsonUtils.toStickerMessage(event.getContent());
+
+                            if (stickerMessage.isThumbnailLocalContent()) {
```

Comment:
As `StickerMessage` is now a `MediaMessage`,  could we merge this C+P code with the code above?